shell/platform/windows/flutter_windows_engine.h

```diff
@@ -293,6 +305,8 @@ class FlutterWindowsEngine {
 
   bool semantics_enabled_ = false;
 
+  bool high_contrast_enabled_ = false;
```

Comment:
That is also a viable option. My thought process behind this choice was that there are a number of other accessibility features that exist but are currently not tracked in Windows, and this would make it more clear what is already implemented. Also, it more neatly mirrors what is already done for semantics. Let me know if you agree with that idea or think storing the flag itself and checking its bits when queried would be preferable.

Reply:
I'm a bit concerned that the number of accessibility fields here will get unwieldy as we add more accessibility features to the Windows embedder.

> there are a number of other accessibility features that exist but are currently not tracked in Windows, and this would make it more clear what is already implemented

That's a good goal, but this class's public API is a stronger declaration of what's supported rather than its private fields.

Overall I don't feel too strongly about this. We can always refactor this later when needed :)

Reply:
Agreed. We can synthesise the flags value anytime we need; individual flags are easier to reason about.